instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -64,6 +64,8 @@ async def redis_get():
 response_hook (Callable) - a function with extra user-defined logic to be performed after performing the request
 this function signature is: def response_hook(span: Span, instance: redis.connection.Connection, response) -> None
 
+sanitize_query True(Default) or False - enable the Redis query sanization
```

Comment:
I'd say Boolean type, and fix naming to sanitization, something like:
`sanitize_query (Boolean) - default True, enable the Redis query sanitization `

Reply:
I've updated the documentation 👍